go/views/index.pug

```diff
@@ -0,0 +1,5 @@
+head
+  title #{.title}
+body
+  p #{.message}
+  button(style="font-size:20px",onclick="location.href='/redirect'") Start journey!
```

Comment:
Why `onclick` when you can use `<a href=`?

Reply:
I liked the idea of it being a button over just a link

Reply:
If you wrap the button with an `<a>` then it's both.

Reply:
Are they not functionally equivalent?

Reply:
No, as `onclick` is not equivalent accessibility wise.